operator/builtin/output/forward/forward_test.go

```diff
@@ -59,3 +59,31 @@ func TestForwardOutput(t *testing.T) {
 		require.Equal(t, newEntry.Resource, e.Resource)
 	}
 }
+
+func TestFlushBufferOnClose(t *testing.T) {
+	cfg := NewForwardOutputConfig("test")
+	received := make(chan []byte, 1)
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		body, _ := ioutil.ReadAll(req.Body)
+		received <- body
+	}))
+
+	cfg.Address = srv.URL
+	ops, err := cfg.Build((testutil.NewBuildContext(t)))
+	require.NoError(t, err)
+
+	forwardOutput, ok := ops[0].(*ForwardOutput)
+	require.True(t, ok)
+
+	newEntry := entry.New()
+	newEntry.Body = "test"
+	newEntry.Timestamp = newEntry.Timestamp.Round(time.Second)
+	err = forwardOutput.Process(forwardOutput.ctx, newEntry)
```

Comment:
I think the context here should actually be `context.Background()`, not using the internal context.